jwst/coron/align_refs_step.py

```diff
@@ -1,45 +1,54 @@
-""" Replace bad pixels and align psf image with target image."""
+"""Replace bad pixels and align psf image with target image."""
 
 from astropy.nddata.bitmask import interpret_bit_flags
 from stdatamodels.jwst import datamodels
 from stdatamodels.jwst.datamodels.dqflags import pixel
 
 from ..stpipe import Step
 from . import imageregistration
-from . median_replace_img import median_replace_img
+from .median_replace_img import median_replace_img
```

Comment:
How did this not trigger any error before? Is this module not used at all?

Reply:
It surprised me too.  Apparently it's not a syntax error:
`In [1]: from jwst.coron import align_refs_step

In [2]: align_refs_step.median_replace_img
Out[2]: <function jwst.coron.median_replace_img.median_replace_img(img_model, box_size, bad_bitvalue)>
`